templates/web/base/around/postcode_form.html

```diff
@@ -1,46 +1,48 @@
 <div id="front-main">
-    <div id="front-main-container">
-        [% UNLESS possible_location_matches %]
-            [% INCLUDE 'around/intro.html' %]
-        [% END %]
+    <div class="container">
```

Comment:
Why is the container element now wrapped in another container element? It now has #front-main-container having a max-width of $container-max-width, and .container also has max-width $container-max-width. So I'm confused as to why both are necessary or what is happening here.

Reply:
The waste template still wraps front-main-container in a `.container`.